### PR TITLE
link to GitHub guide on contributing to a project

### LINKED ISSUE #128: Optional Challenge 1: 404'd
In [lesson 5: GH Pages](https://librarycarpentry.org/lc-git/05-github-pages/index.html) the first optional challenge links to thinkful.com for directions on committing to another's page.  The link 404'd for me today.
I tried searching for the title on thinkful.com, which sent me to a [repo in GitHub](https://github.com/Thinkful/guide-github-pull-request) which looped me back to the same 404 screen.
I do not know how to suggest a better link; this is an important part of the lesson for me at this stage in my learning.

### PR DESCRIPTION
**Pull request by tobyhodges**
Fixes #128 by replacing the broken link to the Thinkful tutorial with a more up-to-date resource on forking, cloning, and pull requests. Hopefully this guide from GitHub is a safer bet in terms of long-term maintenance and availability.

### CODE REVIEW

**Review by ostephens — APPROVED** [review]
Looks good to me. Generally I'd suggest hyperlinks better done with meaningful text rather than just the URL but I see you are just replacing the existing link in the same style. This might be an issue the lesson wants to consider more generally

**Comment by ostephens** [comment]
Ideally this would be reviewed by a lesson maintainer before it's merged @LibraryCarpentry/lc-git-maintainers

**Comment by tobyhodges** [comment]
> Generally I'd suggest hyperlinks better done with meaningful text rather than just the URL but I see you are just replacing the existing link in the same style.

Honestly, I think the exercise text could be generally expanded and improved. I think that would be a great contribution for someone else in the community to make. I've tried to keep my changes here to a minimum just to remove the broken link, and will open a `good-first-issue`/`help-wanted` issue to suggest polishing the exercise.

File: _episodes/05-github-pages.md
Comment (zkamvar) [inline _episodes/05-github-pages.md:122]: Another point to add to https://github.com/LibraryCarpentry/lc-git/pull/131#pullrequestreview-1147654948: [long links are an accessibility issue](https://webaim.org/techniques/hypertext/link_text#link_length):

> Remember that blind users cannot visually skim through lengthy links. They must listen to the entire text. Some get frustrated with long link text and impatiently move on to the next link if they cannot understand the purpose of the link after the first few words.

```suggestion
> Try following the steps in [GitHub's quickstart guide for contributing to projects](https://docs.github.com/en/get-started/quickstart/contributing-to-projects)
```

File: _episodes/05-github-pages.md
Comment (emcaulay) [inline _episodes/05-github-pages.md:124]: I'd like to make some grammatical / style changes to this line. I recommend a period (.) to end the sentence after "new fork" and then have a new sentence that starts "Start instead at the section 'Cloning a fork.'
I would also remove the phrase "if you like" because I think directions should be clear and avoid ambiguity. "if you like" seems to leave it up to me to make a decision that i don't feel as a novice confident enough to make.

**Review by emcaulay — CHANGES_REQUESTED** [review]
Hi @tobyhodges , I have a minor stylistic revision for you to consider.

**Comment by tobyhodges** [comment]
Great suggestions, thanks @emcaulay. I have updated the wording and hope this is ready to merge now.

**Review by emcaulay — APPROVED** [review]

**Comment by emcaulay** [comment]
Thank you so much for the work to make this improvement!

**Pull request merged**
